### PR TITLE
fix(header): 11720-remove-about-link-from-header

### PR DESCRIPTION
**Pull request by vkozio**

### CODE REVIEW

**Comment by IlyaIzr** [comment]
I'd expect to do that _after_ about mode is implemented

File: src/RoutesWrap.tsx [hunk 1]
```diff
@@ -41,19 +41,7 @@ type CommonRoutesFeaturesProps = {
...
 const afterChatContent = (loginFeature: boolean) => {
-  return (
-    <div className={s.afterChatContentWrap}>
-      <a
-        href="https://www.kontur.io/portfolio/disaster-ninja/"
-        target="_blank"
-        className={s.link}
-        rel="noreferrer"
-      >
-        <Text type="heading-m">{i18n.t('about')}</Text>
```
Comment (propakov) [inline src/RoutesWrap.tsx:52]: if you remove translations - please also remove them from `src/core/localization/translations/en/common-messages.json`

File: src/RoutesWrap.tsx [hunk 2]
```diff
@@ -41,19 +41,7 @@ type CommonRoutesFeaturesProps = {
 };
 
 const afterChatContent = (loginFeature: boolean) => {
-  return (
-    <div className={s.afterChatContentWrap}>
-      <a
-        href="https://www.kontur.io/portfolio/disaster-ninja/"
-        target="_blank"
-        className={s.link}
```
Comment (propakov) [inline src/RoutesWrap.tsx:49]: .link class from Main.module.css can be removed

File: src/RoutesWrap.tsx (hunk 1 above)
Comment (propakov) [reply src/RoutesWrap.tsx:52]: en and es both

**Pull request closed without merging**